pyoptsparse/pySNOPT/pySNOPT.py

```diff
@@ -376,7 +375,7 @@ def __call__(
                 lenrw = minWorkArrayLength + 200 * (ncon + nvar)
                 self.setOption("Total real workspace", lenrw)
 
-            cw = np.empty((lencw, 8), "c")
+            cw = np.empty((lencw, 8), dtype="|S1")
```

Comment:
Can you explain what this change does?

Reply:
I am not entirely sure.. I think it's the dtype for a single string character? I had to make this change together with `intent(inplace)` since I did not want these things returned, only the `obj`. This change came from the official SNOPT python interface on GitHub.